.github/workflows/ci.yml

```diff
@@ -27,8 +27,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install -r tests/requirements.txt
-          pip install -e .
+          pip install -U pip
+          pip install -U wheel
+          pip install --prefer-binary -r tests/requirements.txt
+          pip install --prefer-binary -e .
```

Comment:
The `--prefer-binary` is optional and can be removed. It chooses binary of source-distribution. But for some reason it was stubbornly selecting the source-distribution and complaining that it couldn't locate some deps (hdf5?).

But if you look at the netCDF4 1.6.3 files (here https://pypi.org/project/netCDF4/1.6.3/#files) and open any wheel with a zip archive manager, and you should see the system libraries there like hdf5, etc.